configs/platforms/windows-2012r2-x64.rb

```diff
@@ -9,10 +9,12 @@
   plat.add_build_repository "http://nexus.delivery.puppetlabs.net/service/local/nuget/nuget-pl-build-tools/"
 
   # We don't want to install any packages from the chocolatey repo by accident
+  plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe update -y chocolatey"
   plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe sources remove -name chocolatey"
 
   plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe install -y mingw-w64 -version 5.2.0 -debug"
   plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe install -y Wix310 -version 3.10.2 -debug -x86"
+  plat.provision_with "cmd.exe /C C:/ProgramData/chocolatey/bin/choco.exe install msvc.2017-win8.1.sdk.en-us -y --cache-location=\"C:\\msvc\""
```

Comment:
Executed this way, is the `cmd.exe /C` necessary?  Or given it's in cygwin can use something like `--cache-location=/cygdrive/c/msvc`?